qcore/simulation_structure.py

```diff
@@ -82,3 +82,33 @@ def get_hf_acc_dir(sim_root):
 def get_hf_bin_path(sim_root):
     return os.path.join(get_hf_acc_dir(sim_root), 'HF.bin')
 
+
+# yaml
+def get_fault_yaml_path(sim_root, fault_name=''):
+    """
+    manual: Albury_VM_home_melodypzhu_Albury_new_bench_Data_VMs_Albury-h0p4_EMODv3p0p4_190105/fault_params.yaml
```

Comment:
What does manual / auto mean in this case?

Can we use simplified examples for the description? I feel like this isn't actually clarifying what the script is doing.

Reply:
I actually think in general there should not be an example in the docstring at all and it should just be a description of what the function does. 

E.g. 
```
"""Gets the fault params yaml for the specified simulation. 

Note: For the manual workflow set fault_name to None as the fault params are stored directly in the simulation directory.
"""
```

Also I don't think the parameters need to be specified in this case as they are pretty self explanatory. If you want to specify them, then thats perfectly fine, but give a description instead of a example in my opinion.

E.g.
```
sim_root: Path to the `Runs` directory for the automated workflow, or the path the simulation directory for the manual workflow
fault_name: Name of the fault for automated workflow, None for manual workflow
```

Also on that note, I think the default value for fault_name should be None as in my opinion that is much clearer.
And then just add an extra line
```
fault_name = '' if fault_name is None else fault_name
```

Reply:
Very good example, thanks. I have updated the comments